bin/beaker-hostgenerator

```diff
@@ -5,4 +5,8 @@ $LOAD_PATH.unshift(lib_dir)
 
 require 'beaker-hostgenerator'
 
-BeakerHostGenerator::CLI.new.execute!
+cli = BeakerHostGenerator::CLI.new
+
+if not cli then
```

Comment:
Just a quick note, I had no idea what I was talking about up there--this particular change (which is undone in new commit) came about as a result of my misinterpretation of a stack trace I was seeing.